virttest/bootstrap.py

```diff
@@ -955,7 +956,6 @@ def bootstrap(options, interactive=False):
                 LOG.debug("Module %s loaded", module)
 
     LOG.info("")
-    LOG.info("VT-BOOTSTRAP FINISHED")
-    LOG.debug("You may take a look at the following online docs for more info:")
-    LOG.debug(" - http://avocado-vt.readthedocs.org/")
-    LOG.debug(" - http://avocado-framework.readthedocs.org/")
+    LOG.info("vt-bootstrap finished")
```

Comment:
As some teams already adjusted, let's keep this new way. I attempted to minimize the impact by inverting the logic, but the damage is done, let's not break it again and next time please try to avoid breakages if possible (unless it requires obscure workarounds...)